modules/manage/pages/cluster-maintenance/disk-utilization.adoc

```diff
@@ -173,14 +173,16 @@ Compaction, or key-based retention, saves space by retaining at least the most r
 
 When compaction is configured, topics take their size from `compacted_log_segment_size`. The `log_segment_size` property does not apply to compacted topics.
 
-Setting a `segment.bytes` size on a topic applies whether the topic is compacted or not, and the `max_compacted_log_segment_size` property applies to compacted topics regardless of any other properties. The `max_compacted_log_segment_size` property controls how many segments are merged together. For example, if you set `segment.bytes` to 128 MB, but leave `max_compacted_log_segment_size` at 5 GB, then you get 128 MB segments when they're written, but up to 5 GB segments after compaction.
+When compaction executes, one or more segments are merged into one new compacted segment. The old segments are deleted. The size of the initial segments are controlled by `segment.bytes`.  The `max_compacted_log_segment_size` property controls how many segments are merged together. For example, if you set `segment.bytes` to 128 MB, but leave `max_compacted_log_segment_size` at 5 GB, then fresh segments are 128 MB but merged segments may grow to up to 5 GB after compaction.
 
 Redpanda periodically performs compaction in the background. The compaction period is configured by the cluster property xref:reference:cluster-properties.adoc#log_compaction_interval_ms[log_compaction_interval_ms].
 
 Keep in mind that very large segments delay, or possibly prevent, compaction. A very large active segment cannot be cleaned up or compacted until it is closed, and very large closed segments require significant memory and CPU to process for compaction. Very small segments increase the frequency of processing for applying compaction and resource limits. To calculate an upper limit on segment size, divide the disk size by the number of partitions. For example, if you have a 128 GB disk and 1000 partitions, the upper limit of the segment size is `134217728`. Default is `1073741824`.
 
 For details about how to modify cluster configuration properties, see xref:./cluster-property-configuration.adoc[Cluster configuration].
 
+For further information on how compaction functions, see xref:./compaction-tuning.adoc[Compaction tuning].
```

Comment:
- Any chance we could use a simpler word such as "works" in place of "functions"?
- Nit, I noticed that the title in the left nav is "Manage Topic Compaction" but the page title is "Compaction Settings" (and the filename is compaction-tuning which matches the link text). Just want to double check if we need those elements to more closely match?

Reply:
I'll make that change on the wording, and I updated the description, I'll update the filename too.